mypy/stubtest.py

```diff
@@ -243,6 +259,60 @@ def _belongs_to_runtime(r: types.ModuleType, attr: str) -> bool:
         )
 
 
+IGNORED_DUNDERS = frozenset({
+    # Very special attributes
+    "__weakref__",
+    "__slots__",
+    "__dict__",
+    "__text_signature__",
+    # Pickle methods
+    "__setstate__",
+    "__getstate__",
+    "__getnewargs__",
+    "__getinitargs__",
+    "__reduce_ex__",
+    "__reduce__",
+    # typing implementation details
+    "__parameters__",
+    "__origin__",
+    "__args__",
+    "__orig_bases__",
+    "__mro_entries__",
+    "__forward_is_class__",
+    "__forward_module__",
+    "__final__",
+    # isinstance/issubclass hooks that type-checkers don't usually care about
+    "__instancecheck__",
+    "__subclasshook__",
+    "__subclasscheck__",
+    # Dataclasses implementation details
+    "__dataclass_fields__",
+    "__dataclass_params__",
+    # ctypes weirdness
+    "__ctype_be__",
+    "__ctype_le__",
+    "__ctypes_from_outparam__",
+    # Two float methods only used internally for CPython test suite, not for public use
+    "__set_format__",
```

Comment:
I'd prefer float ones to be in the typeshed allowlist

Reply:
(Hypothetical, perhaps not very likely...)

What if a non-typeshed user of stubtest has a float subclass (let's call it `FloatSub`), and they suddenly start getting weird error messages about `__set_format__` being missing from the stub for `FloatSub`?

Reply:
Only about as likely as some user for stubtest unwittingly choosing `__set_format__` as a method name, then getting confused about the lack of checking ;-)

Reply:
Heh, alrighty, I'll take them out!